AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -232,7 +234,10 @@ - (void)_updateVisibleNodeIndexPaths
   _allPreviousIndexPaths = allCurrentIndexPaths;
   
   _currentRangeMode = rangeMode;
-  _didUpdateCurrentRange = NO;
+  // Reset the current range mode only if the range controller comes visible
+  if (ASInterfaceStateIncludesVisible(selfInterfaceState)) {
```

Comment:
Is this actually needed?  Looks safe but I think not needed because this is also checked as part of the condition above.